src/plugins/saved_objects_management/public/management_section/object_view/components/not_found_errors.tsx

```diff
@@ -44,23 +44,29 @@ export const NotFoundErrors = ({ type }: NotFoundErrors) => {
   };
 
   return (
-    <EuiCallOut
-      title={
-        <FormattedMessage
-          id="savedObjectsManagement.view.savedObjectProblemErrorMessage"
-          defaultMessage="There is a problem with this saved object"
-        />
-      }
-      iconType="alert"
-      color="danger"
-    >
-      <div>{getMessage()}</div>
-      <div>
-        <FormattedMessage
-          id="savedObjectsManagement.view.howToFixErrorDescription"
-          defaultMessage="If you know what this error means, go ahead and fix it &mdash; otherwise click the delete button above."
-        />
-      </div>
-    </EuiCallOut>
+    <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
```

Comment:
How do I get this page to render? Try as I might, I can't seem to modify a saved object enough to show the errors.

Reply:
nm, I forced it through hacking the code (which I undid again :-))